src/Nelmio/Alice/Fixtures/Parser/Methods/Base.php

```diff
@@ -102,17 +102,20 @@ protected function processIncludes($data, $filename)
     /**
      * @param array $data
      * @param array $includeData
+     * @return array
      */
     protected function mergeIncludeData($data, $includeData)
     {
-        foreach ($includeData as $class => $fixtures) {
-            if (isset($data[$class])) {
-                $data[$class] = array_merge($fixtures, $data[$class]);
+        $newData = $includeData;
```

Comment:
This is per my request earlier - I feel like this variable name makes the intention much more clear.